### ISSUE TITLE
ID 257224: [Supporting the platform -Simon – Edit Panel]: Contents present in “Repeat options” under "Grafana Play Home/Edit panel" blade are getting truncated after resizing the page to 200% Zoom.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66364]
### **User Experience:**  
Users who have vision issues will be affected if controls are truncated after increasing the screen zoom to 200%. They will not be able to access the controls present on the screen and might get disoriented.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/) Sign in with valid credentials. "Home" page appears. 
2. Navigate the "Feature Showcase pane" heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate the page "Edit" page which appears. 
5. Navigation to "Repeat options" button present on the right pane and expand it. 
6. Verify and observe the issue.

**Actual Result:** 
Issue: Contents present in “Repeat options” under "Grafana Play Home/Edit panel" blade are getting truncated after resizing the page to 200% Zoom.  

**Expected Result:**
Solution: When resizing the page to 200%, all the contents should be clearly visible and should not get truncated under "Repeat options".

### **Severity:**
`SEV2`

### **Area:**
`PanelEdit`
